CMakeLists.txt

```diff
@@ -257,6 +258,9 @@ if(FLATBUFFERS_BUILD_GRPCTEST)
   if(CMAKE_COMPILER_IS_GNUCXX)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-parameter -Wno-shadow")
   endif()
+  INCLUDE_DIRECTORIES(${GRPC_INSTALL_PATH}/include)
```

Comment:
If these env vars are not specified, we get.. `/include` ?

Reply:
Yes. I'll add `if` to bail out if the vars are not defined.